COMPILING.md

```diff
@@ -110,11 +111,14 @@ the base sets which are not yet included in the base set information files.
 To avoid this behaviour, disable GRFCodec (and NFORenum) in CMake cache
 (`GRFCODEC_EXECUTABLE` and `NFORENUM_EXECUTABLE`).
 
-## Developers settings
-
+## Compilation settings
 You can control some flags directly via `CXXFLAGS` (any combination
 of these flags will work fine too):
 
+### Dedicated server
+- `-DOPTION_DEDICATED=ON`: compiles the dedicated server version, without libraries unneeded in a headless environment
```

Comment:
Not only that I am rewriting some other parts I noticed this is wrong. This chapter is about `CXXFLAGS`, while `-DOPTION_DEDICATED` is a CMake flag :) I have a PR incoming that also includes this PR, fixing this :)